data/json/professions.json

```diff
@@ -1841,6 +1906,7 @@
     "requirement": "achievement_kill_100_monsters",
     "description": "The boss always said he could rely on you to pull through on the tough jobs.  Shame he got himself smoked.  No problem; the world's always got a place for someone with your kind of talents.",
     "points": 4,
+    "proficiencies": [ "prof_knives_familiar", "prof_unarmed_familiar" ],
```

Comment:
Oh, I didn't add knife skills because of the swithcblade, but because knifes are a common tool of gangsters and the like (Even if the profession spawns without one), and because of their skills, but I can remove it if you like.

Reply:
Criminals *do* know that they can end up in prison, what if gangster has both shivs & knife basic prof🤔

Reply:
> Criminals *do* know that they can end up in prison, what if gangster has both shivs & knife basic prof🤔

Good argument! I will add it to them, I think the convict assassin should also receive the proficiency for the same reason, so I will add it to them, too.